dashboards/iris/activity_view.json

```
@@ -372,7 +372,7 @@
       "timeFrom": null,
       "timeRegions": [],
       "timeShift": null,
-      "title": "Elapsed time (months) by VO and Month",
+      "title": "Elapsed time (months) by Site and Month",
```

This one should use the terminology "Activity". It's the other chart on this page that's inconsistent.

```suggestion
      "title": "Elapsed time (months) by Activity and Month",
```